examples/tty_conv.c

```diff
@@ -17,7 +21,7 @@
  ***************************************/
 static void echoOff(int fd, int off)
 {
-    struct termios tty;
+    struct termio tty;
```

Comment:
termio is deprecated, termios is the successor.
If you have an implementation, which does not support current standards, make an exception for this implementation, but not for everything else.

Reply:
The code uses a few ioctls, namely, TCGETA, TCSETAF, and TCSETAW, which makes the whole program non-portable.
I suppose when the code uses a syscall interface like this, it should rather use kernel headers and the data types defined there.  Or better use a standard interface like termios(3) instead.